Updated L.Utils method creation to ES6 syntax

AdityaAshvin opened the pull request:
Fixes #628 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [x] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

AdityaAshvin commented:
Hey @sashadev-sky , how to resolve the failing checks?

sashadev-sky commented:
@AdityaAshvini thank you!

It looks like the branch you made the commit on doesn't exist (see here: https://github.com/publiclab/Leaflet.DistortableImage/pull/729/commits/a0d10af8f19ccb1f72c60f41a8bdc4e3fd4e5a2c) so the Travis build will always remain queued.

Try this:
    1. `$ git merge a0d10af8f19ccb1f72c60f41a8bdc4e3fd4e5a2c`
    2. `$ git checkout main`
    3. `$ git remote add upstream https://github.com/publiclab/Leaflet.DistortableImage.git`
    4. `$ git pull upstream main`
    5. `$ git push -f origin main`
    6. `$ git checkout <branch_name>`
    7. `$ git rebase main`
    8. `$ git push -f origin <branch_name>`

AdityaAshvin commented:
> @AdityaAshvini thank you!
> 
> It looks like the branch you made the commit on doesn't exist (see here: [a0d10af](https://github.com/publiclab/Leaflet.DistortableImage/commit/a0d10af8f19ccb1f72c60f41a8bdc4e3fd4e5a2c)) so the Travis build will always remain queued.
> 
> Try this:
> 1. `$ git merge a0d10af8f19ccb1f72c60f41a8bdc4e3fd4e5a2c`
> 2. `$ git checkout main`
> 3. `$ git remote add upstream https://github.com/publiclab/Leaflet.DistortableImage.git`
> 4. `$ git pull upstream main`
> 5. `$ git push -f origin main`
> 6. `$ git checkout <branch_name>`
> 7. `$ git rebase main`
> 8. `$ git push -f origin <branch_name>`

@sashadev-sky Tried these steps, but the Travis build remains queued.

sashadev-sky commented:
@AdityaAshvin I fixed the problem by repushing your commit to trigger a new build, and updated some travis configurations so hopefully it doesn't happen in the future. Thanks for your work, this is ready to merge!

AdityaAshvin commented:
> @AdityaAshvin I fixed the problem by repushing your commit to trigger a new build, and updated some travis configurations so hopefully it doesn't happen in the future. Thanks for your work, this is ready to merge!

Thanks @sashadev-sky. Looking forward to many more contributions😀
